tests/testthat/_snaps/conditions/stop-error-no-trace.txt

```diff
@@ -0,0 +1,4 @@
+Error in h() : !
+Calls: <Anonymous> ... withCallingHandlers -> withVisible -> eval -> eval -> f -> g -> h
```

Comment:
I think this is the error you see when a vignette has an error, so that might suggest if we can entrace in a vignette we can get better errors?

Reply:
Good idea. I'll see if I can adapt knitr for this.